Lib/test/test_site.py

```diff
@@ -309,8 +309,10 @@ def test_no_home_directory(self):
             del environ['APPDATA']
 
             user_base = site.getuserbase()
-            self.assertTrue(user_base.startswith('~' + os.sep),
-                            user_base)
+
+            if sys.platform != "vxworks":
+                self.assertTrue(user_base.startswith('~' + os.sep),
+                                user_base)
```

Comment:
Would you mind to test the expected user_base value? Something like:

```
self.assertEqual(user_base, sys.prefix)
```

Reply:
Added.